devel/zanata-cli/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                zanata-cli
+version             4.4.5
+categories          devel
+platforms           darwin
+license             LGPL-2.1+
+maintainers         {l2dy @l2dy} openmaintainer
+description         Zanata Java command-line client
+long_description    ${description}
```

Comment:
A better long description is probably in order. I think it's important that you can tell what something is, at least in a vague way, by reading the long description.